pymc3/distributions/distribution.py

```diff
@@ -565,30 +565,30 @@ def draw_values(params, point=None, size=None):
         # specified in the point. Need to find the node-inputs, their
         # parents and children to replace them.
         leaf_nodes = {}
-        named_nodes_parents = {}
-        named_nodes_children = {}
+        named_nodes_descendents = {}
```

Comment:
While you are changing this, what about pulling this out and making it a free-standing function?  This block of code is awfully long, and I think the reader/maintainer doesn't need to know the details wouldn't it be ok to have

`named_nodes_ancestors, named_nodes_descendents = build_node_tree(symbolic_params)`